wandb/__init__.pyi

```diff
@@ -1140,7 +1140,7 @@ def link_model(
         ValueError: if name has invalid special characters
```

Comment:
Looks like this didn't get updated. CC @dmitryduev - shouldn't there be a CI check that fails in this case?

@estellazx can you update the return types for these functions in `wandb/__init__.template.pyi` and run `python tools/generate_stubs.py`? These are the global functions that correspond to calling the run methods you modified on the global run.

Reply:
there is, and curiously it passed: https://github.com/wandb/wandb/actions/runs/14738749615/job/41371292732?pr=9763#step:8:130

gotta look into why